src/model/asset.js

```diff
@@ -21,7 +21,7 @@ class Asset {
                 }
 
                 if (query === '') {
-                    url += '&limit=100';
+                    url += '&limit=999';
```

Comment:
The listing of 999 assets seems to take a few seconds under the best conditions. (test it [here](https://superheroes.frontify.com/projects/668/test?q=file&scope=set_recursively), I've invited you). There's no visual feedback. Let's fix that by adding a loading indicator.

Reply:
We already have a loading indicator when clicking on 'Colors' or 'Typography' and we should be able to reuse that 😄

Reply:
Done :-)